sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -746,4 +746,16 @@ public static RetryContext getRetryContext(CosmosDiagnostics cosmosDiagnostics)
             return null;
         }
     }
+
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static void  initializeAllAccessors() {
+        CosmosClient.initialize();
+        CosmosAsyncClientEncryptionKey.initialize();
+        CosmosAsyncContainer.initialize();
+        CosmosAsyncDatabase.initialize();
+        CosmosClientBuilder.initialize();
```

Comment:
Is there any dependency among these initializers?

Reply:
No depdnencies - only thing we need to ensure is that any new accessor adds initialization to one of teh bridgexxx classes